modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -448,13 +448,6 @@ public void postProcessParameter(CodegenParameter parameter) {
 
         operations.put("hasSomeFormParams", hasSomeFormParams);
 
-        // Add additional filename information for model imports in the services
-        List<Map<String, Object>> imports = (List<Map<String, Object>>) operations.get("imports");
-        for (Map<String, Object> im : imports) {
-            im.put("filename", im.get("import"));
-            im.put("classname", im.get("classname"));
-        }
-
```

Comment:
NOTE: This is not actually needed for the fix.
However, after the the changes in this PR this `filename` property is no longer used in the templates. And this is not reliable anyway in the case of inline models. So I just did this cleanup rather than accumulating it as tech debt.

Apparently, this might break some setups that use externally-provided templates (via `--template-dir`).
I'm not sure what the policy is for such changes - is it safe to push to master? Should there be any changelog covering that? Or is it ok because codegen does not consider such custom properties a part of a public contract?

Open to suggestions here!

Reply:
I suggest to leave it there, and remove it later for a major release

Reply:
I'll restore these shortly and add a comment saying it's not used any more